src/BaseFirestoreRepository.spec.ts

```diff
@@ -164,8 +173,15 @@ describe('BaseFirestoreRepository', () => {
           albumsSubColl.orderByAscending('releaseDate').orderByDescending('releaseDate');
         }).toThrow();
       });
+
+      it('must succeed when orderBy* function is called more than once in the same expression with different fields', async () => {
+        const pt = await bandRepository.findById('porcupine-tree');
+        const albumsSubColl = pt.albums;
+        expect(() => {
+          albumsSubColl.orderByAscending('releaseDate').orderByDescending('name');
+        }).toBeTruthy();
```

Comment:
Can you add two tests where you call the same `orderBy` method twice with different fields, such as:

```typescript
expect(() => {
  albumsSubColl.orderByAscending('releaseDate').orderByAscending('name');
}).not.toThrow();
```

```typescript
expect(() => {
  albumsSubColl. orderByDescending('releaseDate'). orderByDescending('name');
}).not.toThrow();
```

Reply:
done!